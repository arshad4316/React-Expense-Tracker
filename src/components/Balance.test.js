import React from 'react';

import renderer from 'react-test-renderer';
import { Balance } from './Balance';

test('snapshot', () =>{
    const component = renderer.create(<Balance/>);

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})