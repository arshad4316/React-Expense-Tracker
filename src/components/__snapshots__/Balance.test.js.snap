// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Array [
  <h4>
    Your Balance
  </h4>,
  <h1>
     0.00
  </h1>,
]
`;
